src/js/controller/bridges.js

```diff
@@ -268,13 +269,14 @@ myApp.controller("BridgesCtrl", [ '$scope', '$rootScope', '$location', 'SettingF
       $scope.send_error = '';
 
       StellarApi.send($scope.destination, $scope.asset.code, $scope.asset.issuer,
-        $scope.asset.amount, $scope.memo_type, $scope.memo, function(err, hash){
+        $scope.asset.amount, $scope.memo_type, $scope.memo, function(err, result){
```

Comment:
We were never using the hash before, so it can be replaced with the full result object.